packages/fxa-auth-server/scripts/test-local.sh

```diff
@@ -16,6 +16,7 @@ node -r ts-node/register ./scripts/gen_keys.js
 node -r ts-node/register ./scripts/gen_vapid_keys.js
 node -r ts-node/register ./scripts/oauth_gen_keys.js
 node -r ts-node/register ../fxa-auth-db-mysql/bin/db_patcher > /dev/null
+node -r ts-node/register ./scripts/update-oauth-db.js
```

Comment:
nit: A name like `oauth-db-patcher` reads a little cleaner to me what this script would do.